### ISSUE TITLE
OpenBLAS 0.3.4 fails to build on x86_64-darwin (in nixpkgs)

### ISSUE THREAD

**Issue by basvandijk** [xianyi/OpenBLAS#1926]
After upgrading openblas from 0.3.3 to 0.3.4 in nixpkgs [the `x86_64-darwin` job](https://hydra.nixos.org/job/nixpkgs/trunk/openblas.x86_64-darwin) started failing.

If you follow the [build log](https://nix-cache.s3.amazonaws.com/log/vvbxzm8bmi06xb38iv7l59mni1y5rkcl-openblas-0.3.4.drv) you'll see the following errors:

```
clang -O2 -DMAX_STACK_ALLOC=2048 -DEXPRECISION -Wall -m64 -DF_INTERFACE_GFORT  -fPIC -DDYNAMIC_ARCH -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=64 -DMAX_PARALLEL_NUMBER=1 -DVERSION=\"0.3.4\" -march=skylake-avx512 -DASMNAME=_dgemm_kernel_SKYLAKEX -DASMFNAME=_dgemm_kernel_SKYLAKEX_ -DNAME=dgemm_kernel_SKYLAKEX_ -DCNAME=dgemm_kernel_SKYLAKEX -DCHAR_NAME=\"dgemm_kernel_SKYLAKEX_\" -DCHAR_CNAME=\"dgemm_kernel_SKYLAKEX\" -DNO_AFFINITY -DTS=_SKYLAKEX -I.. -DBUILD_KERNEL -DTABLE_NAME=gotoblas_SKYLAKEX -DDOUBLE  -UCOMPLEX -c -DDOUBLE -UCOMPLEX ../kernel/x86_64/dgemm_kernel_4x8_skylakex.c -o dgemm_kernel_SKYLAKEX.o
...
../kernel/x86_64/dgemm_kernel_4x8_skylakex.cclang -c -O2 -DMAX_STACK_ALLOC=2048 -DEXPRECISION -Wall -m64 -DF_INTERFACE_GFORT  -fPIC -DDYNAMIC_ARCH -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=64 -DMAX_PARALLEL_NUMBER=1 -DVERSION=\"0.3.4\" -march=skylake-avx512 -DASMNAME=_cgerc_k_SKYLAKEX -DASMFNAME=_cgerc_k_SKYLAKEX_ -DNAME=cgerc_k_SKYLAKEX_ -DCNAME=cgerc_k_SKYLAKEX -DCHAR_NAME=\"cgerc_k_SKYLAKEX_\" -DCHAR_CNAME=\"cgerc_k_SKYLAKEX\" -DNO_AFFINITY -DTS=_SKYLAKEX -I.. -DBUILD_KERNEL -DTABLE_NAME=gotoblas_SKYLAKEX -UDOUBLE  -DCOMPLEX -UDOUBLE -DCONJ ../kernel/x86_64/../generic/zger.c -o cgerc_k_SKYLAKEX.o
...
../kernel/x86_64/dgemm_kernel_4x8_skylakex.c:1315:30: warning: unused variable 'ymm3' [-Wunused-variable]
                        __m256d ymm0, ymm1, ymm2, ymm3, ymm4, ymm5, ymm6, ymm7;
                                                  ^
...
../kernel/x86_64/dgemm_kernel_4x8_skylakex.c:872:5: error: invalid alignment value
                        ".align 32\n"
                         ^
<inline asm>:26:8: note: instantiated into assembly here
.align 32
       ^
../kernel/x86_64/dgemm_kernel_4x8_skylakex.c:1040:5: error: invalid alignment value
                        ".align 32\n"
                         ^
<inline asm>:18:8: note: instantiated into assembly here
.align 32
       ^
../kernel/x86_64/dgemm_kernel_4x8_skylakex.c:1168:5: error: invalid alignment value
                        ".align 32\n"
                         ^
<inline asm>:11:8: note: instantiated into assembly here
.align 32
       ^
```

**Comment by martin-frbg** [comment]
This is #1902 , already fixed on the develop branch

**Comment by basvandijk** [comment]
Great. Will there be a new release soon? Otherwise I just apply [the patch](https://github.com/xianyi/OpenBLAS/commit/701ea88347461e4c5d896765438dc870281b3834.patch) to our 0.3.4 nixpkgs package.

**Comment by martin-frbg** [comment]
current plan is to release 0.3.5 by the end of this month (see under "Milestones")

**Comment by basvandijk** [comment]
@martin-frbg thanks, that's good to know. I [patched our 0.3.4 package](https://github.com/NixOS/nixpkgs/pull/52566) in the mean time.
